aten/src/ATen/cpu/vec/vec_base.h

```diff
@@ -147,9 +147,9 @@ struct Vectorized {
   // versions GCC/Clang have buggy determinations on whether or not an
   // identifier is odr-used or not, and in any case it's hard to tell if
   // a variable is odr-used or not.  So best to just cut the problem at the root.
-  static constexpr size_type size_T = sizeof(T);  // Workaround to compile with VS2022.
+  //static constexpr size_type size_T = sizeof(T);  // Workaround to compile with VS2022.
```

Comment:
Do you mind deleting this line altogether? And half of the preamble about not supporting C++17 yet?